src/Combinatorics/SimplicialComplexes.jl

```diff
@@ -0,0 +1,353 @@
+##  Properties
+################################################################################
+
+@doc Markdown.doc"""
+    nvertices(K::SimplicialComplex)
+
+Number of vertices of the abstract simplicial complex `K`.
+
+# Example
+```jldoctest
+julia> nvertices(torus())
+7
+```
+"""
+nvertices(K::SimplicialComplex) = pm_object(K).N_VERTICES
+
+@doc Markdown.doc"""
+    facets(K::SimplicialComplex)
+
+Maximal (by inclusion) faces of the abstract simplicial complex `K`.
+"""
+function facets(K::SimplicialComplex)
+    bigobject = pm_object(K)
+    the_facets = Vector{Set{Int}}(bigobject.FACETS)
+    return Polymake.to_one_based_indexing(the_facets)
+end
+
+@doc Markdown.doc"""
+    dim(K::SimplicialComplex)
+
+Dimension of the abstract simplicial complex `K`.
+"""
+dim(K::SimplicialComplex) = pm_object(K).DIM
+
+@doc Markdown.doc"""
+    f_vector(K::SimplicialComplex)
+
+Face vector (number of faces per dimension) of the abstract simplicial complex `K`.
+
+# Example
+```jldoctest
+julia> f_vector(torus())
+3-element Vector{Int64}:
+  7
+ 21
+ 14
+```
+"""
+f_vector(K::SimplicialComplex) = Vector{Int}(pm_object(K).F_VECTOR)
+
+@doc Markdown.doc"""
+    h_vector(K::SimplicialComplex)
+
+H-vector of the abstract simplicial complex `K`.
+
+# Example
+```jldoctest
+julia> h_vector(torus())
+4-element Vector{Int64}:
+  1
+  4
+ 10
+ -1
+```
+"""
+h_vector(K::SimplicialComplex) = Vector{Int}(pm_object(K).H_VECTOR)
+
+@doc Markdown.doc"""
+    betti_numbers(K::SimplicialComplex)
+
+Reduced rational Betti numbers of the abstract simplicial complex `K`.
+
+# Example
+```jldoctest
+julia> betti_numbers(klein_bottle())
+3-element Vector{Int64}:
+ 0
+ 1
+ 0
+```
+"""
+betti_numbers(K::SimplicialComplex) = Vector{Int}(Polymake.topaz.betti_numbers(pm_object(K)))
+
+@doc Markdown.doc"""
+    euler_characteristic(K::SimplicialComplex)
+
+Reduced Euler characteristic of the abstract simplicial complex `K`.
+
+# Example
+```jldoctest
+julia> euler_characteristic(complex_projective_plane())
+2
+```
+"""
+euler_characteristic(K::SimplicialComplex) = pm_object(K).EULER_CHARACTERISTIC
+
+@doc Markdown.doc"""
+    minimal_nonfaces(K::SimplicialComplex)
+
+Minimal non-faces of the abstract simplicial complex `K`.
+
+# Example
+```jldoctest
+julia> K = SimplicialComplex([[1,2,3],[2,3,4]]);
+
+julia> minimal_nonfaces(K)
+1-element Vector{Set{Int64}}:
+ Set([4, 1])
+```
+"""
+minimal_nonfaces(K::SimplicialComplex) = Vector{Set{Int}}(Polymake.to_one_based_indexing(pm_object(K).MINIMAL_NON_FACES))
+
+@doc Markdown.doc"""
+    stanley_reisner_ideal(K::SimplicialComplex)
+
+Stanley-Reisner ideal of the abstract simplicial complex `K`.
+
+# Example
+```jldoctest
+julia> stanley_reisner_ideal(real_projective_plane())
+ideal(x1*x2*x3, x1*x2*x4, x1*x5*x6, x2*x5*x6, x1*x3*x6, x1*x4*x5, x3*x4*x5, x3*x4*x6, x2*x3*x5, x2*x4*x6)
+```
+"""
+function stanley_reisner_ideal(K::SimplicialComplex)
+    n = nvertices(K)
+    R, () = PolynomialRing(ZZ, n)
```

Comment:
This is an unusual way (to me, at least) of ignoring the second return value; normally `_` is used for that, I think?